### ISSUE TITLE
Unable to install fastapi-code-generator with python 3.7

### ISSUE THREAD

**Issue by bluecatchbird** [koxudaxi/fastapi-code-generator#54]
I cannot install fastapi-code-generator as described in the readme on `debian 10` with `python 3.7.3`:

```
$ docker run -i -t --rm python:3.7.3 pip install fastapi-code-generator
ERROR: Could not find a version that satisfies the requirement fastapi-code-generator (from versions: none)
ERROR: No matching distribution found for fastapi-code-generator
```

**Comment by koxudaxi** [comment]
@bluecatchbird 
I'm sorry. fastapi-code-generator don't support Python3.7. 
The code-generator requires Python3.8.
I may support Python3.7 at the user's requests.

## Related issues
https://github.com/koxudaxi/fastapi-code-generator/issues/43

**Comment by koxudaxi** [comment]
@bluecatchbird  
I have released a new version as `0.0.17`
This version supports Python3.7
